bundles/mapping/mapmodule/mapmodule.olcs.js

```diff
@@ -699,6 +699,15 @@ class MapModuleOlCesium extends MapModuleOl {
         const lonlat = { lon: location[0], lat: location[1] };
         return lonlat;
     }
+
+    setTime (date) {
+        this._map3D.time = () => Cesium.JulianDate.fromIso8601(date);
```

Comment:
This should modify the Clock (https://cesium.com/docs/cesiumjs-ref-doc/Clock.html) but it comes in the next ticket/task and isn't yet initiated